js/widgets/fixedToolbar.js

```diff
@@ -77,10 +77,10 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 			if ( tclass && tclass !== "none" ) {
 				// use appropriate slide for header or footer
 				if ( tclass === "slide" ) {
-					tclass = this.element.hasClass( "ui-header" ) ? "slidedown" : "slideup";
+					tclass = this.element.hasClass( "ui-toolbar-header" ) ? "slidedown" : "slideup";
```

Comment:
I know this was not you but can we please update `tclass` to be a real variable name.